what-is-the-ic/1_ic/index.md

```diff
@@ -4,8 +4,6 @@ card: /img/what-is-the-ic/built.png
 cardImageFit: center
 ---
 
-The Internet Computer Protocol software runs on standardized server hardware, which we call *nodes*. Nodes are spread around the world, hosted in numerous independent data centers in order to ensure a high degree of fault tolerance and decentralization. Nodes are grouped together into *subnets*, where each subnet constitutes its own blockchain that makes progress independently of the other subnets. All subnets are connected and directed by the Internet Computer Protocol to form the Internet Computer.
+The Internet Computer Protocol runs on servers with standardized hardware, which we call nodes. Nodes are distributed globally and hosted in independent data centers to ensure fault tolerance and decentralization. Nodes are grouped together into subnets, where each subnet constitutes its own blockchain that makes progress independently of the other subnets, resulting in unprecedented performance and scalability in the blockchain space. All subnets are connected and orchestrated by the Internet Computer Protocol to give rise to the Internet Computer.
```

Comment:
I saw some people on the forum that considered 'standardized' to be a form of centralization - not sure we want to promote this fact. Also, we're trying to move off of standardization